src/sequence_controllers/sc_00_COM_ES1_mission_setup.py

```diff
@@ -9,35 +9,45 @@
 
 
 import krpc
+from src.util.commander.constants import *
 
 
 def main(param):
+    status = SMOOTH_FLYING
 
-    # establish connection
-    conn = krpc.connect(name='Artemis 3')
+    # noinspection PyBroadException
+    try:
+        # establish connection
+        conn = krpc.connect(name='Artemis 3')
 
-    # TODO: load celestial bodies
-    for name, obj in conn.space_center.bodies.items():
-        if name == 'Kerbin':
-            kerbin = obj
-        elif name == 'Mun':
-            mun = obj
+        # TODO: load celestial bodies
+        for name, obj in conn.space_center.bodies.items():
+            if name == 'Kerbin':
+                kerbin = obj
+            elif name == 'Mun':
+                mun = obj
```

Comment:
I'm not familiar with Kerbal - should we check for an `else` condition here? Is it possible that some other item (another planet in the Kerbal solar system perhaps) ends up going through this function?

I leave that to your judgement. Either way, this has my approval for merge.

Reply:
No, unfortunately there's not a better way to get these objects stored in short variables. There's a limited number of uniquely named bodies (< 30), so this loop will take like 0.1 seconds. Plus it's not running more than once.